sql/core/src/test/scala/org/apache/spark/sql/test/SQLTestUtils.scala

```diff
@@ -63,7 +63,13 @@ private[sql] trait SQLTestUtils
    * but the implicits import is needed in the constructor.
    */
   protected object testImplicits extends SQLImplicits {
-    protected override def _sqlContext: SQLContext = self._sqlContext
+    protected override def _sqlContext: SQLContext = self.sqlContext
+
+    implicit class StringToColumn(val sc: StringContext) {
```

Comment:
can you add back that comment from `SQLTestUtils` "This must be duplicated here for binary compatibility ..."